academic_observatory_workflows/database/schema/pubmed/article_additions_2023-01-01.json

```diff
@@ -0,0 +1,1504 @@
+[
```

Comment:
I would just call this pubmed.json and the table pubmed, since it isn't loading sharded data and we will have to re-run the workflow if we want to change the schema.